listenbrainz_spark/recommendations/candidate_sets.py

```diff
@@ -1,3 +1,18 @@
+# This script is responsible to create candidate sets for all users. The generated candidate sets
+# will be given as input to the recommender to assign ratings to the recordings in candidate sets.
+# The general flow is as follows:
+#
+# Last 7 days listens are filtered from mapped_listens_df and is called the mapped_listens_subset_df.
```

Comment:
The listens of the last 7 days are filtered from `mapped_listens_df` and this is called the `mapped_listens_subset_df`.